end_to_end/end_to_end_suite_test.go

```diff
@@ -2522,6 +2522,33 @@ LANGUAGE plpgsql NO SQL;`)
 			assertArtifactsCleaned("20240710143553")
 			testhelper.AssertQueryRuns(restoreConn, "DROP TABLE a; DROP TABLE b; DROP TABLE c; DROP TABLE d; DROP TABLE e; DROP TABLE f; DROP TABLE g; DROP TABLE h;")
 		})
+		It("Will not start next batch after error during resize single data file restore", func() {
+			command := exec.Command("tar", "-xzf", "resources/4-segment-db-single-data-file.tar.gz", "-C", backupDir)
```

Comment:
Why have you taken the test case with 10 tables? And it seems you've loaded it with enormous data volume. Maybe make the resources more lightweight?

Reply:
reduced and simplified